concent_api/api-test.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import datetime
+import random
```

Comment:
Co sądzicie o alfabetycznym sortowaniu importów wewnątrz grup importów ? Teraz jeszcze nie jest trudno coś znaleźć ale jak dojdzie ich więcej to potem już czasami zajmuje to chwilę.

Reply:
Zawsze to jakaś metoda, w przyszłości zacznę stosować, nie wiem czy to jest do zmiany w tym request-cie, no ale niech się @cameel wypowie.

Reply:
Let's not change it here. But yeah, we should be sorting them.

I remember that pylint or pyflakes was pointing this out in earlier projects. Now it just ignores it. I need to review the list of ignored warnings (we had to disable some due to lots of false-positives).